src/oop/app/src/Transporters/cacert.pem

```diff
@@ -0,0 +1,3460 @@
+##
+## Bundle of CA Root Certificates
```

Comment:
Why did you commit this file ?

Reply:
It was need to VERIFY_PEER. Switched off and deleted file.